### ISSUE TITLE
Youtube feeds are no longer updated (beginning a few days ago)

### ISSUE THREAD

**Issue by elsawalker** [nt1m/livemarks#158]
**Add-on version**
Add-on version: 1.19

**Describe the bug**
Youtube feeds are no longer updated (beginning a few days ago).

**To Reproduce**
Reminder to create/use a Youtube feed :
- with Username "xxxxx" : https://www.youtube.com/feeds/videos.xml?user=xxxxx
- with Channel_Id "yyyyy" : https://www.youtube.com/feeds/videos.xml?channel_id=yyyyy

**Expected behavior**
Youtube feeds are regularly updated with the new videos posted.

**Extra context**

**Comment by evilpie** [comment]
Works for me.

**Comment by elsawalker** [comment]
Sorry to insist, but nothing changed, still does not work for me since my first post on Feb 13, 2019.

And what do you mean by : "Make sure Tracking Protection or Content Blocking in Firefox aren't interfering". What should be changed exactly in Tracking Protection or Content Blocking for Livemarks and Youtube Feeds in Firefox, and how ?

I didn't change any of these settings, does it mean you changed something for Youtube Feeds ? Because all my non "Youtube Feeds" are updated.

And still unable to add new Youtube feeds (retried BBC example shown in second post on Feb 14, 2019) ==> got same screen shot !

Thanks for, maybe, a more precise answer.

**Comment by nt1m** [comment]
Hi @elsawalker,

We've had issues with Tracking Protection/Content Blocking interfering with feeds, but it doesn't sound like this is the problem here.

Do you have the same problems with other feeds too ?

Cheers,
Tim

**Comment by elsawalker** [comment]
Hi @nt1m and @evilpie,

No, like I said all other feeds (non Youtube Feeds) don't have that refresh problem.
But I think I resolved the problem (for now, I'll wait a few days to be sure).

Following your answer saying "We've had issues with Tracking Protection/Content Blocking interfering with feeds", I tested different things in Firefox preferences like switching from "Strict" Content Blocking to "Standard", but no difference, "Youtube Feeds" refresh or add still did not work.

I'm pretty sure the problem was solved after I deleted the Youtube entry in "Cookies and Site Data" in Firefox preferences (maybe an old Youtube cookie was the trouble ?).
As I modified some other stuff at the same time, I cannot be 100% sure.

But now "Youtube Feeds" refreshing or adding works again. And my BBCNews example (https://www.youtube.com/feeds/videos.xml?user=bbcnews) displays the proper screen :

![screenshot_2019-03-06_16-15-23](https://user-images.githubusercontent.com/32288544/53892104-f46bfd00-402b-11e9-9d9d-8ab5eb43a181.png)

Thanks @nt1m and @evilpie for the hint that helped me out.
